src/utils/cacheManagement.ts

```diff
@@ -0,0 +1,32 @@
+import { Client, ForumChannel, TextChannel, ThreadChannel } from "discord.js";
+
+export async function updateCache(client: Client) {
+    // Iterate through channels in the cache
+    client.channels.cache.forEach(async (channel) => {
```

Comment:
The update here seems to pull in quite a lot of data.
Almost brute force invalidating all the cache?
Which cases are we trying to cache for? Can we run without?

Reply:
We can limit the amount of data fetched primarily by limiting which channels we target and setting a max nr of messages. On the dev server I didn't see any reason for concern, the process was basically instant, but then again we don't have millions of messages there. We can also increase the interval so it only happens like once per hour or something, I honestly have zero clue as to how long it takes before its cache is wiped and we need to refetch stuff, it's the big drawback of me not having any access to proper logs for debugging 😅

Reply:
And the reason we cache is because if we don't cache messages then the bot can't react to events related to the messages